pylds/base.py

```diff
@@ -71,13 +160,19 @@ def check_if_points_escape_box(u, box_boundaries):
     u_indices : array_like, shape(n, )
         array of True/False bool values if points inside/outside the box
     """
-    x, y = u.T
-    # Escape condition
-    box_x_min, box_x_max = box_boundaries[0]
-    box_y_min, box_y_max = box_boundaries[1]
-    u_indices = (x >= box_x_min) & (x <= box_x_max) & (y >= box_y_min) & (y <= box_y_max)
+    N_dof = u.shape[-1]
+    points_positions = u.T[:int(N_dof/2)]
     
-    return u_indices
+    if len(points_positions) == len(box_boundaries):
+        isin_axis_limits = lambda x, box_axis_limits: (box_axis_limits[0]<=x)&(x<=box_axis_limits[1])
+        f = isin_axis_limits
```

Comment:
Is the purpose of `f` only shortening the name of `isin_axis_limits`?

Reply:
Yep. Renamed function `isin_axis_limits` as `check` though. Thus, line 168 is gone.